### PR TITLE
RED-5091 - Fix pypi upload

### PR DESCRIPTION
**Pull request by matthewbelisle-wf**
Fixing build

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -4,6 +4,16 @@ jobs:
   Publish:
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v2
```
Comment (matthewbelisle-wf) [inline .github/workflows/publish.yml:7]: I totally missed this aspect of the pypi-publish action.

https://github.com/marketplace/actions/pypi-publish

> This GitHub Action has nothing to do with building package distributions. Users are responsible for preparing dists for upload by putting them into the dist/ folder prior to running this Action.

**Review by patkujawa-wf — APPROVED** [review]
:moreyouknow:

**Comment by dancleereman-wf** [comment]
RM +1

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**